src/components/proximities/proximities.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-unused-vars */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Heading2 } from '../typography/headline/h2';
+import { colors } from '../../typography/colors';
+import { BodySecondary } from '../typography/body/body-secondary';
+import './proximities.css';
+import { Separator } from '../separator/separator';
+import { Divider } from '../cards/divider/divider';
+import {
+  IconDefinition,
+  faCartShopping,
+  faGraduationCap,
+  faHospital,
+  faLeaf,
+  faUtensils,
+} from '@fortawesome/free-solid-svg-icons';
+
+const proximitiesData = {
+  proximities: [
+    {
+      type: 'WellRatedRestaurants',
+      description: 'Cabanã Osasco, Salva Burguer, Jin Chef e O Bom do Baião.',
+    },
+    {
+      type: 'GreenAreas',
+      description: '2° Floresta Urbana de Osasco, Parque Chico Mendes e Praça Laurindo de Camargo.',
+    },
+    {
+      type: 'Schools',
+      description: 'E.E. Profº Francisco Casabona, ETEC Professor André Bogasian e Colégio Padre Anchieta.',
+    },
+    {
+      type: 'Colleges',
+      description: 'Universidade Estácio - Polo Osasco, Universidade Anhembi Morumbi e FNJ - Faculdade Nove de Julho.',
+    },
+    {
+      type: 'Hospitals',
+      description: 'Hospital Nossa Senhora de Fátima e Hospital Geral de Carapicuíba..',
+    },
+    {
+      type: 'Markets',
+      description: 'Atacadão, Mercado Moretti Bittencourt e Mercado Municipal de Osasco.',
+    },
+  ],
+};
+enum proximitiesType {
```

Comment:
nome de enum a gente da com letra maiuscula, igual interface e classes
```suggestion
enum ProximitiesType {
```